Geometry/DTGeometry/test/DTGeometryAnalyzer.cc

```diff
@@ -9,6 +9,8 @@
 #include <FWCore/Framework/interface/one/EDAnalyzer.h>
 #include <FWCore/Framework/interface/EventSetup.h>
 #include <FWCore/Framework/interface/ESHandle.h>
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
```

Comment:
@cvuosalo - it would be nice to use the double quotes for all CMSSW includes for consistency.